CDS/scripts/js_utils/datastore/config.js

```diff
@@ -0,0 +1,66 @@
+const path = require('path');
+const fs = require('fs');
+const PropertiesReader = require('properties-reader');
+
+class Config {
+    constructor (configDirectory = '/etc/cds_backend/conf.d', namespaceChar = '_') {
```

Comment:
@fredex42 It won't break. The `namespaceChar` is used to key as a key in the `config` property of this class. There isn't a limit on the number of `_`.